.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+require( '@automattic/eslint-plugin-wpvip/init' );
+
+module.exports = {
+	extends: [
+		'plugin:@automattic/wpvip/recommended',
+		'plugin:@automattic/wpvip/cli',
+		'plugin:@automattic/wpvip/weak-testing',
+		'plugin:@automattic/wpvip/weak-typescript',
+	],
+	rules: {
+		camelcase: 'warn',
+		'jest/no-mocks-import': 'warn',
+		'no-await-in-loop': 'warn',
+		'no-unused-vars': 'warn',
```

Comment:
```suggestion
```

In TypeScript, we need to use `@typescript-eslint/no-unused-vars` instead. `no-unused-vars` has to be disabled.

Ref: https://typescript-eslint.io/rules/no-unused-vars/#how-to-use

Reply:
Resolved in 2b16e73 and 008de55.